src/system/rqgCalculations.ts

```diff
@@ -5,20 +5,38 @@ type LookupTableEntry<T> = {
 };
 
 export class RqgCalculations {
-  public static dexSR(dex: number): number | undefined {
+  public static dexSR(dex: number | undefined): number | undefined {
+    if (dex == null) {
+      return undefined;
+    }
     return this.lookup<number | undefined>(dex, dexSrTable);
   }
 
-  public static sizSR(siz: number): number | undefined {
+  public static sizSR(siz: number | undefined): number | undefined {
+    if (siz == null) {
+      return undefined;
+    }
     return this.lookup<number | undefined>(siz, sizSrTable);
   }
 
-  public static hitPoints(con: number, siz: number, pow: number): number {
-    return con + RqgCalculations.linearMod(siz) / 5 + RqgCalculations.flattenedMod(pow) / 5;
+  public static hitPoints(
+    con: number | undefined,
+    siz: number | undefined,
+    pow: number | undefined
+  ): number | undefined {
+    let hp =
+      con != null && siz != null && pow != null
```

Comment:
Since you need con, siz & pow to calculate hitpoints I assume that you need all of them to have hitpoints. @Moonpile you only looked at con, but I think this makes more sense, do you agree?

Reply:
I don't think you actually need SIZ and POW to have hit points.  They just modify the hit points as determined by CON.  If you don't have either SIZ or POW they just wouldn't be modified.  However, I'm not at all sure what someone without SIZ but **with** hit points would look like.  With SIZ and CON but no POW I would assume undead or a construct.

Reply:
Thinking about this again.  The real check happens in RqgActor line 46 because we want the entire hitPoints property to be undefined (or should it be null on line 55?) if you do not have CON.  If you have CON then you have hitPoints because they're based on CON and only modified by SIZ and POW.  

CON, SIZ, POW works (as the "normal" mode)
CON, no SIZ, POW works
CON, SIZ, no POW works
no CON = no hit points no matter the other two.

As is, this works properly to the best of my understanding of the rules.